README.md

```diff
@@ -5,6 +5,8 @@
 This repository holds the icons and logos for all the brands Home Assistant
 supports.
 
+Please do not submit icons and logos for custom components.
```

Comment:
I don't think this should be in the introduction text of the repository.

Down below it states:

> The `./src` folder contains a folder for each `domain` Home Assistant provides
an integration for.

Home Assistant does not provide custom integrations. However, we could extend that sentence maybe with:

> The `./src` folder contains a folder for each `domain` Home Assistant provides
an integration for. Custom integration domains are not provided by Home Assistant and thus not in this repository.

Reply:
I think some further explanation would help as it seems there are many PR-s related to custom components. The current introduction states what the repo contains but it does not explicitly state what it does **not** contain.

Reply:
It states that already: "icons and logos for all the brands Home Assistant supports."

Custom integration are not supported by Home Assistant.

I'm fine with adding an additional note, but not in the introduction text of the repository (as said before).

Reply:
Your version of the extended sentence is totally fine, I did not mean to resist it but instead explain the need of some additional information. Feel free to add your version and close this PR.